lambdas/static-asset-uploader/index.js

```diff
@@ -291,7 +291,12 @@ function handler(event, context) {
 
         if (event.RequestType === "Delete") {
             console.log(`bucketName: ${bucketName}, staticBucketName: ${staticBucketName}`)
-            return exports.cleanS3Bucket(bucketName)
+            // I know that this is a promise, but please don't return it
+            // If you return it, lambda will kill the node runtime, even if there's
+            // work to do in the event loop still. notifyCFN...() uses a callback pattern
+            // and isn't represented well by the promise this returns. It can be fixed
+            // in the future by moving to nhutch@'s promise-wrapped notifyCFN...()
```

Comment:
can we create and link to an issue to follow up on this? Seems like a potentially sharp edge to be aware of

Reply:
Ah, this is actually fixed in a later commit ( https://github.com/awslabs/aws-api-gateway-developer-portal/pull/222/commits/1c5bbda7239f63af7dcd01c33caf93d3f4260c48 )